alf/data_structures.py

```diff
@@ -124,30 +150,6 @@ def make_action_time_step(time_step: TimeStep, prev_action, first_env_id=0):
         'state'  # state passed to rollout() to generate `action`
     ])
 
-
```

Comment:
why do you remove this function? Seems we still need this.

Reply:
We do not have the old TimeStep, the new TimeStep has 'action' field, so maybe this function can be substituted by time_step._replace(action=action) ?